### PR TITLE
En 1748 improvement of page's group handling (version 4.3.4)

### PR DESCRIPTION
**Pull request by eugeniosant**

### CODE REVIEW

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by joewhite101** [comment]
Retest this please

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by pietrangelo** [comment]
retest this please

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by pietrangelo** [comment]
retest this please

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by pietrangelo** [comment]
retest this please

**Comment by entando-service** [comment]
All is Well

File: plugins/entando-plugin-jpseo/src/main/java/org/entando/entando/plugins/jpseo/aps/system/services/page/SeoPageDAO.java [hunk 1]
```diff
@@ -36,48 +36,50 @@
...
+ *
  * @author E.Santoboni
  */
 public class SeoPageDAO extends PageDAO {
 
-	private static final Logger _logger =  LoggerFactory.getLogger(SeoPageDAO.class);
+    private static final Logger _logger = LoggerFactory.getLogger(SeoPageDAO.class);
+
+    @Override
+    protected PageExtraConfigDOM getExtraConfigDOM() {
```
Comment (joewhite101) [inline plugins/entando-plugin-jpseo/src/main/java/org/entando/entando/plugins/jpseo/aps/system/services/page/SeoPageDAO.java]: A "get" method that creates a new object is a bad pattern. We should rename this to `createExtraConfigDom()`

**Pull request merged**